### ISSUE TITLE
Should we have a "Next" button when performing the OAuth dance

### ISSUE THREAD

**Issue by zregvart** [syndesisio/syndesis-ui#1181]
It doesn't do anything so we might as well remove it?

![selection_306](https://user-images.githubusercontent.com/1306050/32456474-115f84a6-c326-11e7-8859-993a4a72f030.png)

**Comment by gashcrumb** [comment]
Agreed, that next button should go away when oauth is involved...  

I wonder actually if the back/next buttons would be better placed down underneath the form when it's showing instead of up at the breadcrumb bar.

**Comment by deeleman** [comment]
> I wonder actually if the back/next buttons would be better placed down underneath the form when it's showing instead of up at the breadcrumb bar.

+1 to that

And if so, the next button would override the "Connect ..." CTA, or just trigger the same action.
